infrastructure/database.tf

```diff
@@ -41,3 +42,48 @@ resource "azurerm_key_vault_secret" "POSTGRES-HOST" {
   value        = module.postgresql.fqdn
   key_vault_id = module.civil_sdt_key_vault.key_vault_id
 }
+
+
+module "postgresql-v15" {
+  source = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=master"
+
+  providers = {
+    azurerm.postgres_network = azurerm.private_endpoint
+  }
+
+  admin_user_object_id = var.jenkins_AAD_objectId
+  business_area        = "cft"
+  common_tags          = local.tags
+  component            = var.component-V15
+  env                  = var.env
+  pgsql_databases = [
+    {
+      name = "civil_sdt"
+    }
+  ]
+  pgsql_version        = "15"
+  product              = var.product
+  name               = join("-", [var.product, var.component-V15])
+  backup_retention_days = "${var.database_backup_retention_days}"
+}
+
+# Create secret for database user
+resource "azurerm_key_vault_secret" "POSTGRES-USER-V15" {
+  name         = "civil-sdt-POSTGRES-USER-V15"
+  value        = module.postgresql-v15.username
+  key_vault_id = module.civil_sdt_key_vault.key_vault_id
+}
+
+# Create secret for database password
+resource "azurerm_key_vault_secret" "POSTGRES-PASS-V15" {
+  name         = "civil-sdt-POSTGRES-PASS-V15"
+  value        = module.postgresql-v15.password
+  key_vault_id = module.civil_sdt_key_vault.key_vault_id
+}
+
+# Create secret for database host
+resource "azurerm_key_vault_secret" "POSTGRES-HOST-V15" {
+  name         = "civil-sdt-POSTGRES-HOST-V15"
+  value        = module.postgresql-v15.fqdn
+  key_vault_id = module.civil_sdt_key_vault.key_vault_id
+}
```

Comment:
Do the secret names in the values files, e.g. [values.yaml](https://github.com/hmcts/civil-sdt/blob/master/charts/civil-sdt/values.yaml#L25-L30) need to be updated to the V15 ones?